### ISSUE TITLE
Ability to forbid customization / overriding of event properties

### ISSUE THREAD

**Issue by muety** [jhuckaby/Cronicle#458]
## Summary
For additional security, it would be helpful to prevent an already existing event's parameters from being customized in the `run_event_call`.

**Comment by jhuckaby** [comment]
Yes, that is a very good idea.  Thank you.  I'll incorporate that into the next major version.
